samples/cellular/modem_shell/src/link/link_shell_print.c

```diff
@@ -243,8 +243,7 @@ link_shell_map_to_string(struct mapping_tbl_item const *mapping_table,
 	}
 
 	if (!found) {
-		sprintf(out_str_buff,
-			"%d (unknown value, not converted to string)", mode);
+		sprintf(out_str_buff, "%d FAIL", mode);
```

Comment:
`UNKNOWN` would be preferable, but that's too big for some of the buffers. Because this should not even happen, I don't think it makes sense to increase the buffer sizes to accommodate a better string. So, this is ok.

Reply:
Changed to plain number copy to output buffer, increase of the minimum buffer from 8 to 16 bytes, and adding an error log.

Please check the changes. @tokangas, @jhirsi